### PR TITLE
[dashboard] Include ray session name in dashboard snapshot

### PR DESCRIPTION
**Pull request by wuisawesome**
## Why are these changes needed?

This includes the ray session name in the snapshot which makes it easy to differentiate "clusters" across restarts. 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: dashboard/modules/snapshot/snapshot_schema.json [hunk 1]
```diff
@@ -134,9 +134,13 @@
               }
             }
           },
+          "sessionName": {
```
Comment (wuisawesome) [inline dashboard/modules/snapshot/snapshot_schema.json:137]: test case validates this schema change

**Review by joeybai — APPROVED** [review]
data + schema lgtm

**Review by ijrsvt — APPROVED** [review]
LGTM!

File: dashboard/modules/snapshot/snapshot_head.py [hunk 2]
```diff
@@ -14,14 +14,17 @@ def __init__(self, dashboard_head):
...
 
     @routes.get("/api/snapshot")
     async def snapshot(self, req):
         job_data = await self.get_job_info()
         actor_data = await self.get_actor_info()
+        session_name = await self.get_session_name()
         snapshot = {
             "jobs": job_data,
             "actors": actor_data,
+            "session_name": session_name,
```
Comment (joeybai) [inline dashboard/modules/snapshot/snapshot_head.py:27]: schema uses camelCase (`sessionName`), but this is using snake_case. please double check.

**Comment by wuisawesome** [comment]
Things are green, merging.

**Pull request merged**